### PR TITLE
Delete Data SDK and Update Readme

### PR DESCRIPTION
**Pull request by v-yilinhu**
1. Delete file Utilities.cs

2. Update files:
    - i ./Program.cs
    - ii ./ServiceBusWithClaimBasedAuthorization.csproj
    - iii ./README.md

@scottaddie,@AlexGhiondea,@lilyjma,@ramya-rao-a and @annatisch for notification.

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -138,15 +215,43 @@ public static void Main(string[] args)
...
                 RunSample(azure);
             }
             catch (Exception e)
             {
-                Utilities.Log(e.ToString());
+                Console.WriteLine(e.ToString());
             }
         }
+
+        public static void PrintSubscriptionsInTopic(Microsoft.Azure.Management.ServiceBus.Fluent.ISubscription serviceBusSubscription)
```
Comment (ramya-rao-a) [inline Program.cs]: This need not be public right? Also please add similar methods to print queues, topics and the namespace

File: README.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ extensions:
  - Create send authorization rule for queue.
  - Create send and listener authorization rule for Topic.
  - Get the keys from authorization rule to connect to queue.
- - Send a "Hello" message to queue using Data plan sdk for Service Bus.
- - Send a "Hello" message to topic using Data plan sdk for Service Bus.
  - Delete namespace
 
+ More about how to use Azure Service Bus see [here](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/servicebus/Azure.Messaging.ServiceBus).
```
Comment (ramya-rao-a) [inline README.md]:
```suggestion
 For more on how to use Azure Service Bus see the [samples for sending and receiving messages](https://docs.microsoft.com/samples/azure/azure-sdk-for-net/azuremessagingservicebus-samples/).
```

File: Program.cs (hunk 1 above)
Comment (v-yilinhu) [reply Program.cs]: @ramya-rao-a I have updated, please review.

File: Program.cs [hunk 3]
```diff
@@ -54,47 +49,66 @@ public static void RunSample(IAzure azure)
...
+
+                PrintSubscriptionsInTopic(subscription2);
 
                 //=============================================================
                 // Create new authorization rule for queue to send message.
-                Utilities.Log("Create authorization rule for queue ...");
+                Console.WriteLine("Create authorization rule for queue ...");
                 var sendQueueAuthorizationRule = serviceBusNamespace.AuthorizationRules.Define("SendRule").WithSendingEnabled().Create();
-                Utilities.Print(sendQueueAuthorizationRule);
+                StringBuilder builderSendQueueAuthorizationRule = new StringBuilder()
```
Comment (ramya-rao-a) [inline Program.cs]: Please make a separate method for printing rules just like you have for queues and topics

File: Program.cs [hunk 4]
```diff
@@ -54,47 +49,66 @@ public static void RunSample(IAzure azure)
...
+                            .Append("\n\t\t\tName :").Append(right.ToString());
+                }
+
+                Console.WriteLine(builderSendQueueAuthorizationRule.ToString());
 
-                Utilities.Log("Getting keys for authorization rule ...");
+                Console.WriteLine("Getting keys for authorization rule ...");
                 var keys = sendQueueAuthorizationRule.GetKeys();
-                Utilities.Print(keys);
+                StringBuilder builderKeys = new StringBuilder()
```
Comment (ramya-rao-a) [inline Program.cs]: Please make a separate method for printing keys just like you have for queues and topics

Comment (v-yilinhu) [reply Program.cs]: @ramya-rao-a I have updated, please review.

File: Program.cs (hunk 3 above)
Comment (v-yilinhu) [reply Program.cs]: @ramya-rao-a I have updated, please review.

**Review by ramya-rao-a — APPROVED** [review]
Looks good to me

**Comment by v-xuto** [comment]
@ramya-rao-a Does this PR need someone else to review it? If not, please help us merge this PR.

**Comment by ramya-rao-a** [comment]
@jongio should be able to help with merging the PR here

**Comment by v-xuto** [comment]
@jongio Do you have permission for this repo? If so, please help us merge this PR.

**Pull request merged**